src/ralph/settings/base.py

```diff
@@ -287,9 +287,28 @@ def str_to_bool(s: str) -> bool:
         }
     })
 
+REDIS_MASTER_IP = None
+REDIS_MASTER_PORT = None
+
+REDIS_SENTINEL_ENABLED = bool_from_env('REDIS_SENTINEL_ENABLED', False)
+if REDIS_SENTINEL_ENABLED:
+    from redis.sentinel import Sentinel
+
+    # SENTINEL_HOSTS env variable format: host_1:port;host_2:port
+    SENTINEL_HOSTS = os.environ['SENTINEL_HOSTS'].split(';')
```

Comment:
consider naming it `REDIS_SENTINEL_HOSTS`

Reply:
ok